CONTRIBUTING.md

```diff
@@ -86,7 +86,7 @@ a few things you can do to help out:
 
 - **Give us a star.** It may not seem like much, but it really makes a
```

Comment:
Could oyu please make changes to this file over at https://github.com/ory/meta/blob/master/templates/repository/common/CONTRIBUTING.md ? :)

Reply:
Reverted this file.